IdentityCore/src/controllers/MSIDLocalInteractiveController.m

```diff
@@ -79,7 +79,7 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
 
         if (installBrokerResponse)
         {
-            [self promptBrokerInstallWithResponse:installBrokerResponse completionBlock:completionBlock];
+            [self handleBrokerResponse:installBrokerResponse completion:completionBlock];
```

Comment:
Q: will this response ever be non-install broker? I don't see why this was changed from promptBrokerInstall to handleBrokerResponse

Reply:
Or is this meant to be handleBrokerInstallResponse??

Reply:
Changed naming, please take a look.